### DISCUSSION TITLE
摸鱼工具-fkfish

### DISCUSSION THREAD

**Discussion by caibingcheng** [caibingcheng/comment#111] [Announcements]
https://bbing.com.cn/202205/about-fkfish/

**Comment by penndu** [comment]
这个好这个好，通过这个对付领导！

**Comment by caibingcheng** [comment]
> @penndu
> 这个好这个好，通过这个对付领导！

要是可以屏蔽系统热键就好了, 现在还不行, 有权限问题

**Comment by penndu** [comment]
> @caibingcheng
> > @penndu
> > 这个好这个好，通过这个对付领导！
> 
> 要是可以屏蔽系统热键就好了, 现在还不行, 有权限问题

那倒没事，没人会动我的电脑！

**Comment by oopswoo3** [comment]
哈哈哈哈这个好
